src/olympia/activity/migrations/0016_auto_20220826_1055.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.15 on 2022-08-26 10:55
+
+import ipaddress
+from django.db import migrations
+
+
+def backfill_iplog_ip_address_binary(apps, schema_editor):
+    IPLog = apps.get_model('activity', 'IPLog')
+    for log in IPLog.objects.filter(ip_address_binary__isnull=True).iterator():
```

Comment:
Could be done in pure SQL with `UPDATE log_activity_ip SET ip_address_binary = INET6_ATON(ip_address) WHERE ip_address_binary IS NULL;`. Doing it in Python is slower, but the bet is that this slowness is useful to let the replicas catch up. I could also do a backfill command that spawns tasks if we think we need it.

Reply:
https://github.com/mozilla/addons-server/pull/19616/commits/cdd90d91db5a030a4bcf1703484a2e09d33f8aae